src/main/java/lol/game/Team.java

```diff
@@ -60,8 +62,10 @@ public boolean championAttack(int championID, int x, int y) {
     battlefield.visit(x, y, new TileVisitor(){
       @Override public void visitDestructible(Destructible d) {
         attacked[0] = champion.attack(d);
+        sound.attackSound(champion.name());
         if(d.isDead()) {
           battlefield.destroy(d);
+          sound.destroyBuilding();
```

Comment:
Just a small note: at this point, it might not be a building that is destroyed but another champion. Perhaps it is not too important though

Reply:
I will think of an alternative so it is only for buildings